src/layman/layer/rest_workspace_layer.py

```diff
@@ -145,6 +145,7 @@ def patch(workspace, layername):
                                   }) from exp
 
     kwargs['time_regex'] = time_regex
+    kwargs['image_mosaic'] = time_regex is not None if delete_from is not None else None
```

Comment:
`delete_from is not None` -> `delete_from == 'layman.layer.filesystem.input_file'`